sympy/algebras/quaternion.py

```diff
@@ -122,20 +122,21 @@ def from_axis_angle(cls, vector, angle):
 
     @classmethod
     def from_rotation_matrix(cls, M):
-        """Returns the equivalent quaternion of a matrix.
+        """Returns the equivalent quaternion of a matrix. The quaternion will be normalized
+        only if the matrix is special orthogonal (orthogonal and det(M) = 1).
 
-        Parameters
+        Parameters:
         ==========
 
         M : Matrix
-            The matrix from which an equivalent quaternion has to be obtained.
+            Input matrix to be converted to equivalent quaternion. M must be special 
```

Comment:
Your code is failing because of trailing whitespace at this line

Reply:
Thanks for the help! I'm resolving it.